### ISSUE TITLE
Change port of use from 8080

### ISSUE THREAD

**Issue by peytlucas** [dularion/streama#223]
Hi,
I would like to change the port streama is using from 8080 to a different port. How do I go about that?

**Comment by gkiko** [comment]
`java -Dserver.port=8081 -jar streama-0.1.war`

**Comment by peytlucas** [comment]
I am using windows terminal and the installation instructions are specific that I run grailsw.bat run-war. Do I instead use what you just posted?

**Comment by peytlucas** [comment]
I am using windows terminal and the installation instructions are specific that I run grailsw.bat run-war. Do I instead use what you just posted?

Date: Thu, 22 Sep 2016 03:56:19 -0700

**Comment by gkiko** [comment]
So you are running war file from grails console which still uses `java -jar ...` I guess.

In your case same parameters should be enough. [See](http://stackoverflow.com/a/25133262/660408)

**Comment by peytlucas** [comment]
I was using windows command prompt. Let me try grails console then I will get back to you. Thanks

**Comment by peytlucas** [comment]
Can I get full instruction on how to run streama. I am afraid I have tried all the methods posted but still I get no results. When I try using the command you gave me I get the error: "no main manifest attribute". My port 8080 is in use by a system process so I am attempting port 8089, but I keep getting the error: port 8080 is in use. I have altered the configuration files to 8089 and still I keep getting the error port 8080 in use. Where exactly do I need to configure the port?

**Comment by peytlucas** [comment]
Can I get full instruction on how to run streama. I am afraid I have tried all the methods posted but still I get no results. When I try using the command you gave me I get the error: "no main manifest attribute". My port 8080 is in use by a system process so I am attempting port 8089, but I keep getting the error: port 8080 is in use. I have altered the configuration files to 8089 and still I keep getting the error port 8080 in use. Where exactly do I need to configure the port?

Date: Thu, 22 Sep 2016 04:14:30 -0700
